test/e2e/e2e.go

```diff
@@ -469,8 +469,15 @@ func deleteClusterExternalContainer(containerName string) {
 	}, 5).Should(gomega.HaveLen(0))
 }
 
-func updateNamespace(f *framework.Framework, namespace *v1.Namespace) {
```

Comment:
`updateNamespace` was used only from EIP tests from what I can tell,
can we leave the older func as is for nonUDN tests if needed ?
we can add your new util and invoke that only from UDN tests/
I assume not all EIP tests need to use the UDN label....